app/api/v2/views/questions_views.py

```diff
@@ -0,0 +1,73 @@
+from flask import jsonify, Blueprint, request, make_response
+from ..models.questions_models import Questions
+from ..models.meetups_models import Meetup
+
+
+v1_question_blueprint = Blueprint('questions', __name__, url_prefix='/api/v1')
+
+questions = Questions()
+meetups = Meetup()
+
+
+@v1_question_blueprint.route('/questions', methods=['POST'])
+def post_question():
+    """ endpoint for creating meetup"""
+
+    data = request.get_json()
+    if not data:
+        return jsonify({"message": "Data set cannot be empty"})
+
+    if not all(field in data for field in ["title", "content"]):
+        return jsonify({"status": 400, "message": "Please fill in all the required input fields"}), 400
+
+    postedBy = data.get('postedBy')
+    meetup_id = data.get('meetup_id')
+    title = data.get('title')
+    content = data.get('content')
+
+    question = questions.post_question(
+        postedBy, meetup_id, title, content)
+
+    return question
+
+
+@v1_question_blueprint.route('/questions', methods=['GET'])
+def getall():
+    """ endpoint to fetch all questions """
+
+    data = questions.getall_questions()
```

Comment:
rename the variable data to be more descriptive according to the contents it contains

Reply:
adjusted as required